flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -717,9 +717,13 @@ var A320_Neo_UpperECAM;
                                 message: "EXCESS CAB ALT",
                                 level: 3,
                                 flightPhasesInhib: [1, 2, 3, 4, 5, 7, 8, 9, 10],
+                                page: "PRESS",
                                 isActive: () => (
                                     !Simplane.getIsGrounded() &&
-                                    this.getCachedSimVar("PRESSURIZATION CABIN ALTITUDE", "feet") > 10000
+                                    this.getCachedSimVar("L:A32NX_PRESS_CABIN_ALTITUDE", "feet") > 9550 &&
+                                    this.getCachedSimVar("L:A32NX_PRESS_CABIN_ALTITUDE", "feet") > (this.getCachedSimVar("L:A32NX_DEPARTURE_ELEVATION", "feet") + 1000) &&
+                                    this.getCachedSimVar("L:A32NX_PRESS_CABIN_ALTITUDE", "feet") > (this.getCachedSimVar("L:A32NX_OVHD_PRESS_LDG_ELEV_KNOB", "feet") + 1000
+                                        || this.getCachedSimVar("L:A32NX_PRESS_AUTO_LANDING_ELEVATION", "feet") + 1000)
```

Comment:
From an FWC perspective this is actually just a single signal that is received through the SDACs from the CPCUs.

As such, in order to be ready for the FWS rework, I would suggest simply moving this logic to your Rust CPC and exposing an individual boolean LVar (e.g. `A32NX_PRESS_EXCESSIVE CAB ALT`). At that point I can then just hook up the FWS to that, rather than having to then move this logic into the CPC myself.